src/lizaalert/courses/models.py

```diff
@@ -295,7 +295,7 @@ class ProgressStatus(models.TextChoices):
     )
 
     def __str__(self):
-        return f"Lesson {self.lesson.title}: {self.user.username}"
+        return f"Lesson {self.lesson.title}: {self.user.username} Progress: {self.get_userlessonprogress_display()}"
```

Comment:
не нужно делать дополнительные запросы

```suggestion
        return f"Lesson {self.lesson_id}: {self.user_id} Progress: {self.get_userlessonprogress_display()}"
```